src/emr_local.ts

```
@@ -318,16 +338,33 @@ export class EMRLocalEnvironment {
     // "source=${env:HOME}${env:USERPROFILE}/.aws,target=/home/hadoop/.aws,type=bind"
     // Also make adding environment credentials optional...they could get exposed in logs
 
-    const dockerfilePath = this.context.asAbsolutePath(
-      path.join("templates", "pyspark.dockerfile")
-    );
-    const dockerfile = fs.readFileSync(dockerfilePath).toString();
+    let dockerfilePath;
+
+    if (getEmrMajorVersion(release) === 5 || getEmrMajorVersion(release) === 6) {
```

EMR 5 should raise an error because it's not supported?